xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-server/xwiki-platform-search-solr-server-data/src/main/resources/xwiki/conf/schema.xml

```diff
@@ -840,12 +840,14 @@
     </fieldType>
 
     <!-- Czech -->
-    <fieldType name="text_cz" class="solr.TextField" positionIncrementGap="100">
+    <fieldType name="text_cs" class="solr.TextField" positionIncrementGap="100">
       <analyzer> 
         <tokenizer class="solr.StandardTokenizerFactory"/>
         <filter class="solr.LowerCaseFilterFactory"/>
         <filter class="solr.StopFilterFactory" ignoreCase="true" words="lang/stopwords_cz.txt" />
```

Comment:
Maybe the best will be let the type definition of text_cz untouched. It is based on the definition from Solr (Lucene) which use cz for Czech language and add a comment to the dynamic field that cs for XWiki means cz for Solr. If the Solr team fix it and change cz to cs, the person doing upgrade of Solr in XWiki will remove the comment and repair the dynamic field definition to include new changes from Solr. If the Solr team don't fix it, the type definition will remain same as in Solr examples which I think is cleaner.